src/core/managers/internal/ToastManager/ToastManager.ts

```diff
@@ -105,18 +106,18 @@ export class ToastManager {
   }
 
   private async updateTransactionToastsList(toastList: ToastsSliceType) {
-    const { transactions: sessions, account } = this.store.getState();
+    const { transactions, account } = this.store.getState();
```

Comment:
inverse question: why not save here sessions since at line 120 we do transactions[toast.toastId];